spec/paths/enterprise@report@group-report@.yaml

```diff
@@ -0,0 +1,49 @@
+get:
+  summary: Get a list response of group reports
+  description: Returns a list of group reports
+  operationId: getGroupReportList
+  produces:
+    - application/json
+  parameters:
+    - name: start_date
+      in: query
+      description: Start date for the group report
+      type: string
+      format: date
+      required: true
+    - name: end_date
+      in: query
+      description: End date for the group report
+      type: string
+      format: date
+      required: true
+    - name: group_name
+      in: query
+      description: Group name for the group report
+      type: string
+      required: true
+  responses: 
+    '200':
+      description: successful operation
+      schema:
+        type: object
+        properties:
+          count:
```

Comment:
group-report and device-report API's does not support pagination yet.
The response should be something like.
[
    {
        "group_name": "offline device group 1",
        "active_devices": 0,
        "inactive_devices": 11,
        "registered_devices": 11,
        "new_devices": 0,
        "disabled_devices": 0
    },
   {...},
   ...
]